desktop/src/splash.html

```diff
@@ -5,7 +5,7 @@
       html {
         width: 100%;
         height: 100%;
-        background: url('../../public/img/homescreen-icon-144x144.png') center center no-repeat;
+        background: url('../../public/img/media.png') center center no-repeat;
```

Comment:
Why did you revert this?

Reply:
Look above in the collapsed comment, I really really do not like that. It looks crappy in comparison to showing the `media.png`.

Reply:
Look at this comparison:

<img width="1440" alt="screen shot 2018-05-30 at 10 10 32" src="https://user-images.githubusercontent.com/7525670/40725979-0e3512be-6425-11e8-892f-471c95a86b06.png">

<img width="1437" alt="screen shot 2018-05-30 at 16 18 20" src="https://user-images.githubusercontent.com/7525670/40726016-241fa17a-6425-11e8-9f15-01e264de871a.png">

The second one looks so much more like a start screen to me?